Sortable.js

```diff
@@ -1220,6 +1233,10 @@
 						differentLevel = dragEl.parentNode !== el,
 						scrolledPastTop = _isScrolledPast(target, axis === 'vertical' ? 'top' : 'left');
 
+					if (lastEl !== target) {
+						lastEl = target;
```

Comment:
Why is this set here? If options.swap is enabled, wouldn't this code never run?

Reply:
Thanks for reviewing! To be honest, I just tried to replicate the changes that RubaXa made, and I think I copied this in without putting much thought into it. I agree, it looks like it can be removed.